frontend/src/components/common/EditorDialog.tsx

```diff
@@ -333,13 +345,10 @@ function DocsViewer(props: {docSpecs: any}) {
   }
 
   return (
-    docSpecs.error ?
-      <Empty>
-        Error getting documentation!
-        Please make sure the YAML is valid and has the kind and apiVersion set.
-      </Empty>
-      : docs === null ?
-        <Loader />
+    docs === null && docsError === null ?
+      <Loader />
+      : !_.isEmpty(docsError) ?
+        <Empty><Typography color="secondary" align="center">{docsError}</Typography></Empty>
```

Comment:
Hmm, you shouldn't need the `typography/secondary` elem here, right? Should all be in the `<Empty>` component.

Reply:
@joaquimrocha i added the Typography inside Empty element because right now our Empty Component doesn't have the facility to pass color from outside because its hardcoded there.And in this case I need to pass the red color in case of errors

Reply:
Ah, but you're using the secondary color. I think you shoud use the `error` color.
Maybe a quick patch just passing the color down to the empty would be better than wrapping this component.